### DISCUSSION TITLE
DArk mode

### DISCUSSION THREAD

**Discussion by enzovici** [ragapp/ragapp#56] [Q&A]
Hi, I can't figure out how to apply .dark class to the UI. Can you help?

**Comment by thucpn** [comment]
I guess you are talking about the .dark class in the global.css file. It is used to define custom CSS variables for dark mode.

To apply dark mode, you can install next-themes and use it as follows:
[Dark Mode with Next.js](https://ui.shadcn.com/docs/dark-mode/next)
